dali/pipeline/pipeline_test.cc

```diff
@@ -403,8 +403,8 @@ DALI_SCHEMA(DummyPresizeOp)
 TEST_F(PipelineTestOnce, TestPresize) {
   const int batch_size = 1;
   const int num_thread = 1;
-  const bool pipelined = true;
-  const bool async =  true;
+  const bool pipelined = false;
+  const bool async =  false;
```

Comment:
Presizing is not supported for Executor 2.0, which can be foricbly enabled with env. `DALI_USE_EXEC2=1`.